### PR TITLE
Prac 05 code review feedback

### PR DESCRIPTION
**Pull request by Eamon-C**
Check for convention breaking or efficiency improvements.

@EricaFinlay

### CODE REVIEW

File: prac_05/state_names.py [hunk 1]
```diff
@@ -4,15 +4,17 @@
...
 while state_code != "":
-    if state_code in CODE_TO_NAME:
+    try:
         print(state_code, "is", CODE_TO_NAME[state_code])
-    else:
-        print("Invalid short state")
-    state_code = input("Enter short state: ")
+    except KeyError:
+        print("Invalid input!")
+    state_code = input("Enter short state: ").upper()
```
Comment (EricaFinlay) [inline prac_05/state_names.py]: Nicely written

**Review by EricaFinlay — COMMENTED** [review]
Looks great

**Review by EricaFinlay — COMMENTED** [review]
All looks good.

**Review by EricaFinlay — COMMENTED** [review]
Looks good

**Review by EricaFinlay — COMMENTED** [review]
I can't see docstring comments for the functions.  I hope that I am looking at the right version.  Are they missing?

**Comment by EricaFinlay** [comment]
Thanks Eamon.  All finished.

**Comment by Eamon-C** [comment]
> 
Comments have been added 👍

**Pull request merged**